etc/ReplayOfflineConfiguration.py

```diff
@@ -37,7 +37,7 @@
 # 355559 - 2022 pp at 13.6 TeV (1h long, 300 bunches)
 # 356005 - 2022 pp at 13.6 TeV (1h long, 600 bunches - ALL detectors included)
 # 359060 - 2022 cosmics - Oberved failures in Express (https://cms-talk.web.cern.ch/t/paused-jobs-for-express-run359045-streamexpress/15232)
-setInjectRuns(tier0Config, [359762])
+setInjectRuns(tier0Config, [325174, 327527])
```

Comment:
I'll remove one of them for ppCollisions run 360886

Reply:
To be honest I don't really see the point of testing a ppCollision run with this configuration, e.g. the scenarios are all set for HI collisions only (maybe we should actually revisit this choice). So either we change back the "standard" scenarios to the pp ones (and in that case we would be testing pp data with pp configuration...it does not seem the goal of this replay), or we leave the scenarios as they are now and we only test HI data.

But maybe I misunderstood the discussion today at the x-coord meeting, @mandrenguyen could you help me understand?

Reply:
Let me also add that in terms of GTs, in case we go for a replay on pp data, the best would be then to have two separate replays for each, as the GTs would have to be different for 2018 and 2022.

Reply:
ok so two different replays makes sense to me :) (sorry i had missed this in the discussion!)
@germanfgv please do not add the pp run in this replay then!